src/ontology/edits/oeo-shared.omn

```diff
@@ -1642,6 +1650,9 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/810"
         <http://purl.obolibrary.org/obo/RO_0000087> some OEO_00040011
     
     
+Class: OEO_00000144
```

Comment:
Should this be deleted?

Reply:
No, it indicates that this class has to be moved to oeo-shared.

Reply:
Okay, I resolved this. However, it implied to move some further classes to oeo-shared.